bundle/libraries/local_path.go

```diff
@@ -80,18 +74,28 @@ func IsLibraryLocal(dep string) bool {
 	return IsLocalPath(dep)
 }
 
-func IsLocalRequirementsFile(dep string) (string, bool) {
-	dep, ok := strings.CutPrefix(dep, "-r ")
-	if !ok {
-		return "", false
+var PipFlagsWithLocalPaths = []string{
+	"-r",
+	"-e",
+}
+
+func IsLocalPathInPipFlag(dep string) (string, string, bool) {
+	for _, flag := range PipFlagsWithLocalPaths {
+		dep, ok := strings.CutPrefix(dep, flag+" ")
```

Comment:
`dep` aliases the input. It works because of `:=` but is error-prone. Recommend to use a different name.